scripts/mbedtls_framework/c_wrapper_generator.py

```diff
@@ -207,15 +237,15 @@ def _skip_function(self, function: FunctionInfo) -> bool:
             return True
         return False
 
-    _FUNCTION_GUARDS = {
+    _function_guards = {
```

Comment:
I missed that earlier: since this is now mutable, it should be an instance field, set in the class's construtor, instead of a class field that's created when the class is defined.